drivers/scheduler/k8s/k8s.go

```diff
@@ -1551,6 +1551,38 @@ func (k *k8s) createCRDObjects(specObj interface{}, timeout, retryInterval time.
 	return nil, nil
 }
 
+func (k *k8s) DecommissionNode(n node.Node) error {
+	k8sOps := k8s_ops.Instance()
+	pods, err := k8sOps.GetPodsByNode(n.Name, "")
+	if err != nil {
+		return &scheduler.ErrFailedToDecommissionNode {
+			Node:          n,
+			Cause:         fmt.Sprintf("Failed to get pods on the node: %v. Err: %v", n.Name, err),
+		}
+	}
+	if err = k8sOps.DrainPodsFromNode(n.Name, pods.Items, defaultTimeout, defaultRetryInterval); err != nil {
+		return &scheduler.ErrFailedToDecommissionNode {
+			Node:          n,
+			Cause:         fmt.Sprintf("Failed to drain pods from node: %v. Err: %v", n.Name, err),
+		}
+	}
+
+	if err = k8sOps.CordonNode(n.Name, defaultTimeout, defaultRetryInterval); err != nil {
+		return &scheduler.ErrFailedToDecommissionNode {
+			Node:          n,
+			Cause:         fmt.Sprintf("Failed to cordon node: %v. Err: %v", n.Name, err),
+		}
+	}
+	if err = k8sOps.AddLabelOnNode(n.Name, "px/enabled", "remove"); err != nil {
```

Comment:
`px/enabled` is a Portworx specific label. It cannot reside in drivers/scheduler/k8s/k8s.go

This needs to happen in the k8s sched-ops inside drivers/volume/portworx/